components/dashboard/src/start/start-workspace-options.ts

```diff
@@ -42,8 +42,8 @@ export namespace StartWorkspaceOptions {
                 };
             }
         }
-        if (params.get(StartWorkspaceOptions.AUTOSTART) === "true") {
-            options.autostart = true;
+        if (params.get(StartWorkspaceOptions.AUTOSTART)) {
+            options.autostart = params.get(StartWorkspaceOptions.AUTOSTART) === "true";
```

Comment:
this is so it sets `false` when the argument is passed but is not set to `true`. I.e. to distinguish between `false` and `undefined`.